AnnoyancesFilter/sections/popups.txt

```diff
@@ -121,6 +121,11 @@ myflixerhd.ru#$#.modal-backdrop { display: none !important; }
 myflixerhd.ru#$#body { overflow: auto !important; }
 facens.br##.facens__unbouncer--overlay
 facens.br##.facens__unbouncer
+digitalvidya.com#$?##page-container > div#et-boc:has( > div.et-l > div[data-da-area="ouibounce_modal"]) { display: none !important; }
+digitalvidya.com#$?##page-container > div#et-boc:has( > div.et-l > div[data-da-area="single_modal"]) { display: none !important; }
```

Comment:
You blocked some useful popups. For example, `Register Now`.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/108315826-0de0c200-71cd-11eb-87e8-8c57860f5e9a.png)

</details><br/>

Try these rules:
```
digitalvidya.com#$?##page-container div[data-da-area="ouibounce_modal"] { display: none !important; }
digitalvidya.com#$?##page-container div[data-da-area="single_modal"] { display: none !important; }
```

Reply:
Are you sure? All buttons works fine on my end
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/68852737/108332124-82712c00-71e0-11eb-8afb-815850dbdfaa.png)</details><br/>

Reply:
I'm sure.
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/108332856-530eef00-71e1-11eb-9924-cba823684def.png)

</details><br/>